src/tiled/layerdock.h

```diff
@@ -77,6 +80,22 @@ private slots:
 };
 
 /**
+ * Delegate for drawing an eye icon in LayerView when the layer is visible.
+ */
+class LayerVisibilityDelegate: public QItemDelegate
```

Comment:
Since this class is not used anywhere else than in `layerdock.cpp`, please move it entirely into the cpp file.